src/evergreen/metrics/buildmetrics.py

```diff
@@ -279,5 +385,16 @@ def __str__(self):
             timeout_pct=self.pct_tasks_timed_out,
             system_failure_count=self.system_failure_count,
             system_failure_pct=self.pct_tasks_system_failure,
+            total_display_tasks=self.total_display_tasks,
```

Comment:
Since this is part of creating human readable string of the object, these should also be added to the string above.

Reply:
These are already there. Do you mean add them to the as_dict method?

Reply:
I mean the string on line 359 - 370 should include these values.

Reply:
fixed